rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] kindly make book and person accessible too according to project requirement